_includes/people.html

```diff
@@ -52,16 +52,15 @@
 
     {% if roles or user.title %}
       <p class="people-description">
+         <p class="people-description-question">
+        Role in OLS:  <p class="people-description-question">
         {% if user.title %}
-        <p class="people-description-question">
-          Role in OLS:
-          <a class="card-toggle">{{ user.title }} <i class="fa fa-angle-down"></i></a>
+          {{ user.title }}
+        {% endif %}
+        {% if user.title %}
```

Comment:
```suggestion
        {% if roles %}
```

Reply:
I was going to point this out. Thanks, @bebatut. 🤝